psalm.xml

```diff
@@ -10,5 +10,10 @@
     <issueHandlers>
         <LessSpecificReturnStatement errorLevel="info" />
         <MoreSpecificReturnType errorLevel="info" />
+        <InternalMethod>
+            <errorLevel type="suppress">
+                <referencedMethod name="ParagonIE_Sodium_Core_Util::store64_le" />
```

Comment:
See https://github.com/paragonie/paseto/issues/111 point [1]: IMHO `ParagonIE_Sodium_Core_Util` should be a isolated package